app/src/androidTest/java/ch/epfl/sdp/location/LocationServiceTest.java

```diff
@@ -116,8 +122,9 @@ public void obtainNullWithoutPermissions() throws Throwable {
     }
 
     @Test
-    public void canStopAggregator() {
+    public void canStopAggregator() throws Throwable {
         mActivityRule.getActivity().getService().onProviderDisabled(LocationManager.GPS_PROVIDER);
         // TODO: This test should check the effects of the operation
```

Comment:
TODO : done ? or to do ?

Reply:
Well spotted. That test now just checks that there are no exceptions. It should be enhanced in the future